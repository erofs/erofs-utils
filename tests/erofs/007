#!/bin/sh
# SPDX-License-Identifier: GPL-2.0+
seq=`basename $0`
seqres=$RESULT_DIR/$(echo $0 | awk '{print $((NF-1))"/"$NF}' FS="/")

# get standard environment, filters and checks
. "${srcdir}/common/rc"

cleanup()
{
	cd /
	rm -rf $tmp.*
}

# remove previous $seqres.full before test
rm -f $seqres.full

# real QA test starts here
echo "QA output created by $seq"

[ -z "$lz4_on" ] && \
	_notrun "lz4 compression is disabled, skipped."

${PWD}/src/badlz4 >> $seqres.full 2>&1 || _fail "--> lz4 test FAILED"

echo Silence is golden
status=0
exit 0
