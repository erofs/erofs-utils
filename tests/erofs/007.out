QA output created by 007
Silence is golden
